packages/teleterm/src/ui/logger.ts

```diff
@@ -1,13 +0,0 @@
-import Logger, { DefaultService } from 'teleterm/logger';
-import { ElectronGlobals } from 'teleterm/types';
-
-export default Logger;
-
-export function initLogger(globals: ElectronGlobals) {
-  const settings = globals.mainProcessClient.getRuntimeSettings();
-  if (settings.dev) {
-    Logger.init(new DefaultService());
-  } else {
-    Logger.init(globals.loggerService);
-  }
```

Comment:
What impact does the removal of this function have on logger behavior in dev mode? As far as I understand, we'll be always using `Logger.init(globals.loggerService)`?

Reply:
Yes.
Note that logic was placed in the `ui/logger`, so in theory it should work that way only for the UI. IMO it could be confusing, why these logs weren't collected?